src/View/TwigView.php

```diff
@@ -179,10 +180,11 @@ protected function generateHelperList()
      * @param string $viewFile Template file.
      * @param array  $data     Data that can be used by the template.
      *
+     * @throws \Exception
      * @return string
      */
     // @codingStandardsIgnoreStart
-    protected function _render($viewFile, $data = array())
+    public function _render($viewFile, $data = array())
```

Comment:
Seems I changed the visibility of the method, which I should not.
Will fix it as soon as I can.